### DISCUSSION TITLE
language configuration

### DISCUSSION THREAD

**Discussion by liamzhang66** [avitorio/outstatic#272] [Ideas]
Firstly, this project is exactly what I was looking for. Compared to other headless CMS solutions, it stores data in a way that satisfies me the most. It’s amazing. Thank you to the developers. Secondly, would it be possible to add a language configuration for the admin panel, allowing the replacement of all field labels via an external file? This could be helpful for users whose native language is not English.

**Comment by avitorio** [comment]
Hey @linuszhang1993, localization is something I want to add to Outstatic eventually. For now we are staying with english, but I'm hoping to add translation settings in the future.
